README.md

```diff
@@ -42,7 +42,7 @@ This tool makes use of `tracing` library for log purposes. For this reason set
 ## Example
 
 ```bash
-cargo run -- --db-path ../pathfinder/mainnet.sqlite --start-block 632917 --end-block 632917 --svg-out "histogram.svg"
+cargo run --release -- --rpc-url https://starknet-mainnet.public.blastapi.io/rpc/v0_7 --start-block 632917 --end-block 632917 --svg-out "histogram.svg"
```

Comment:
Is this a public endpoint that people will be able to hit if they just run this command?

Reply:
The endpoint is known public starknet rpc server and everybody can use it